### PR TITLE
Create weekly-update.2015-05-29.md

### PR DESCRIPTION
**Pull request by Gioyik**
CC #82

### CODE REVIEW

**Comment by yosuke-furukawa** [comment]
io.js v2.2.0 has a problem related npm.  https://github.com/nodejs/io.js/issues/1850

We should append the change log about v2.2.1 https://github.com/nodejs/io.js/blob/master/CHANGELOG.md#2015-06-01-version-221-rvagg

**Comment by Gioyik** [comment]
The new release was a few minutes after I created the PR. I'm going to update it.

**Comment by Gioyik** [comment]
Update done!

Thanks @yosuke-furukawa

File: weekly-updates/weekly-update.2015-05-29.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# io.js 2.2 releases
+This week we had two io.js release [v2.2.0](https://iojs.org/dist/v2.2.0/) and [v2.2.1](https://iojs.org/dist/v2.2.1/), complete changelog can be found [on GitHub](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md).
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-05-29.md]: Typo, release**s**. Also, the changelog URL points to the old org name.

**Comment by bnb** [comment]
:+1: LGTM.

**Comment by yosuke-furukawa** [comment]
merged! please publish @julianduque

File: weekly-updates/weekly-update.2015-05-29.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# io.js 2.2 releases
+This week we had two io.js releases [v2.2.0](https://iojs.org/dist/v2.2.0/) and [v2.2.1](https://iojs.org/dist/v2.2.1/), complete changelog can be found [on GitHub](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md).
```
Comment (tianp) [inline weekly-updates/weekly-update.2015-05-29.md:2]: Changelog URL is incorrect, it supposed to https://github.com/nodejs/io.js/blob/master/CHANGELOG.md

**Pull request merged**